### PR TITLE
Check if we see at least 2 frameworks

### PR DESCRIPTION
**Pull request by janisz**
## High-level description

Do not fail when there are more than 2 frameworks installed.

## Corresponding DC/OS tickets (required)

  - [DCOS-54487](https://jira.mesosphere.com/browse/DCOS-54487) 

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-mergebot label Ready For Review
:wave: @drewvanstone @br-lewis @makkes @orsenthil   Can you review, please?

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py [hunk 1]
```diff
@@ -681,7 +681,7 @@ def validate_state(zip_state):
     assert isinstance(zip_state, zipfile.ZipExtFile)
     state_output = gzip.decompress(zip_state.read())
     state = json.loads(state_output)
-    assert len(state["frameworks"]) == 2, "bundle must contains information about frameworks"
+    assert len(state["frameworks"]) > 1, "bundle must contains information about frameworks"
```
Comment (drewvanstone) [inline packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: /bundle must contains/bundle must contain/

**Comment by janisz** [comment]
:wave: @drewvanstone @br-lewis @makkes  Can you review, please?

**Comment by janisz** [comment]
@mesosphere-mergebot sync

**Pull request merged**